CHANGES

```diff
@@ -9,6 +9,11 @@
 
  Changes between 1.1.1 and 3.0.0 [xx XXX xxxx]
 
+  *) Added OPENSSL_info() to get diverse built in OpenSSL data, such
```

Comment:
Change to built-in otherwise it reads funny.

Reply:
I'll never get the hang of when to and when not to use dashes...  Fixing

Reply:
(and we use "built in" all over the place, it seems.  Still fixing my stuff here, though

Reply:
The only other thing I can think of is the pesky windows backslashes, when building a path

Reply:
> I'll never get the hang of when to and when not to use dashes

Neither will I.

Reply:
Ah, directory-file separator.  Sure, and that will fix the same problem for VMS.

Reply:
Also, list separator character may be useful (we use LIST_SEPARATOR internally in a few places)

Reply:
Okie, both added